inc/thorvg.h

```diff
@@ -1183,6 +1183,15 @@ class TVG_EXPORT Picture final : public Paint
      */
     Result viewbox(float* x, float* y, float* w, float* h) const noexcept;
 
+    /**
+     * @brief Gets the information about the preserveAspectRatio attribute of the loaded SVG picture.
+     *
+     * @warning Please do not use it, this API is not official one. It could be modified in the next version.
+     *
+     * @BETA_API
+     */
+    Result preserveAspectRatio(bool* preserve) const noexcept;
```

Comment:
suggest:
bool aspectRatio() const noexcept;

Reply:
I've chosen the longer name, since this api returns only the info, whether the aspect ratio is preserved or not. It doesn't return the value of the preserveAspectRatio attribute.

In svg files the possible preserveAspectRatio values are 'xMinYMin meet', 'xMidYMid slice', ..., 'none' 
In tvg we support 'none' and only one of the rest - so we force uniform scaling, without going into details regarding xMin/Max etc. 

that's why I thought it may be confusing if the api is called 'aspectRatio', but I changed the name as you suggested

Reply:
@mgrudzinska Hi, two reasons I suggested.
1. API naming consistency. You can check other getter apis.
2. Picture is the abstracted class. This API shouldn't be designed only for SVG.
3. Actually, I don't  know which scenario of this apis is useful so far (It would be better if you mention the *REAL*l user scenario where it's necessary) but In the end I think we might need the AspectRatio setter/getter options with an universal way.

Reply:
@hermet please take a look at #1117 
it's more general case, so if in the future we'd like to support other preserveAspectRatio attributes, than the api will be still valid. the version from this pr is very limited.
if #1117 is not what you meant by 'an universal way' please let me know, I'll try to change the prs accordingly.

The real use case for this api is, that I need it in efl :) otherwise it's probably as useful as the viewbox api is.

Reply:
@mgrudzinska  why efl needs this?